.github/ISSUE_TEMPLATE/3-api-ref-docs-problem.md

```diff
@@ -0,0 +1,54 @@
+---
+name: "\U0001F4D7 Open an issue regarding the Node.js API reference docs"
+about: Let us know about any problematic API reference documents
+title: "doc: "
+labels: doc
+---
+
+# 📗 API Reference Docs Problem
+
+<!------------------------------------------------------------------------------
+Thank you for wanting to make nodejs.org better!
+
+This template is for issues with the Node.js API reference docs.
+
+For more general support, please open an issue in
+our help repo at “https://github.com/nodejs/help”.
+
+For the issue title, enter a one-line summary after “doc: ”.
+The “✍️” signifies a request for input. If unsure, do the best you can.
+
+If you found a problem with nodejs.org beyond the API reference docs, please
+open an issue in our website repo at “https://github.com/nodejs/nodejs.org”.
+------------------------------------------------------------------------------->
+
+<!--
+Version: output of “node -v”
+Platform: output of “uname -a” (UNIX), or version and 32 or 64-bit (Windows)
+Subsystem: if known, please specify affected core module name
+-->
+
+- **Version**: ✍️
+- **Platform**: ✍️
+- **Subsystem**: ✍️
+
+## Location
+
+_Section of the site where the content exists_
+
+Affected URL(s):
+- https://nodejs.org/api/✍️
+
+## Problem description
+
```

Comment:
may be:
 - [ ] missing / incomplete doc
 - [ ] doc is confusing
 - [ ] actual behavior not aligned with doc

Reply:
Although I see value in allowing people to self-categorize the problem, I have a feeling that, if this list were complete, it would be too long. Perhaps we could add this later once we see what the frequent uses of this template are.